src/Components/Navbar.js

```diff
@@ -0,0 +1,61 @@
+import { useCart } from "../Contexts";
+import { Searchbar } from "./Searchbar";
+import { Link,NavLink } from "react-router-dom";
+export const Navbar = ({searchText,setSearchText}) => {
+  const {
+    state: { cartItems },
+    dispatch
+  } = useCart();
+
+  function totalCartItems(cartItems) {
+    const array = cartItems.filter((product) => product.existsInCart);
+    return array.length;
+  }
+
+  return (
+    <div>
+      <nav className="nav-wrapper-3">

```

Comment:
You can work on your navbar styling as it appears consistently on each page - remove underline from logo and product anchor tags.